graphql-schemas/schema.yml

```diff
@@ -1506,6 +1506,15 @@ confs:
   - { name: name, type: string, isRequired: true }
   - { name: text, type: string, isRequired: true }
 
+- name: TerraformRepo_v1
+  fields:
+  - { name: account, type: AWSAccount_v1, isRequired: true }
```

Comment:
For my understanding, I saw it in the design doc that each repo will need to be able to target multiple AWS account, how come this is only tied to one?

Reply:
That's a good point. This was done to allow for bootstrapping of each terraform repo. A tenant could target multiple AWS accounts though by duplicating the terraform repo file and then targeting other AWS accounts. They can even re-use the existing Git repository and just change project paths. The only constraint is that each of terraform repos must have a unique name to avoid tfstate key file collisions.

Reply:
Thank you for the explanation. So from App Interface perspective, we can't really tell which AWS account a bootstrapped repo really targets, we'd go in the repo and look for `./stage-aws-account-name/main.tf
` etc?

Reply:
No, from App Interface you would be able to tell by looking at the terraform repo file, here's an example:

```yaml
---
$schema: /aws/terraform-repo-1.yml

account:
    $ref: /aws/app-int-example-01/account.yml

name: plan_a
repository: <my_repo>
ref: <my_ref>
projectPath: plan_a
```

Each repo would have a file like this in A-I so you could distinguish which repo is being applied to which account.

Reply:
Right but what if the repo is applying to more than one account?